src/core/mac/mac_frame.cpp

```diff
@@ -836,7 +836,7 @@ uint8_t Frame::SkipAddrFieldIndex(void) const
 
     default:
         ExitNow(index = kInvalidIndex);
-        break;
+        OT_UNREACHABLE_CODE(break;);
```

Comment:
nit: in some cases, I see:

- `OT_UNREACHABLE_CODE(break;);`

In others, I see:

- `OT_UNREACHABLE_CODE(break;)`

We should be consistent. I would prefer a single semi-colon and after the closing parent, like:

- `OT_UNREACHABLE_CODE(break);`.

Thoughts?

Reply:
Done. The original code looked like this:
```
OT_UNREACHABLE_CODE(
    break;
);
```
but make pretty compacted it to one line.